src/main/java/com/google/sps/data/QuestionAndChoices.java

```diff
@@ -0,0 +1,25 @@
+package com.google.sps.data;
+
+import java.util.List;
+
+/**
+ * Class to store a question, which includes a string question and several mutiple choice choices of
+ * type T
+ */
+public final class QuestionAndChoices<T> {
+  private String question;
+  private List<T> choices;
+
+  public QuestionAndChoices(String question, List<T> choices) {
```

Comment:
Eventually it probably makes sense for this to be <T extends Choice> where Choice is an interface that both promo/career choices implement.